pypolygames/tests/test_zoo.py

```diff
@@ -30,10 +61,11 @@ def test_models(model_name) -> None:
     input_data = torch.zeros([1] + feature_size, device=torch.device("cpu"))
     outputs = model.forward(input_data)
     assert list(outputs["v"].shape) == [1, 1]
-    assert list(outputs["pi"].shape) == [1] + action_size
+    assert list(outputs["pi_logit"].shape) == [1] + action_size
     # loss
     multi_counter = utils.MultiCounter(root=None)
-    pi_mask = torch.ones(outputs["pi"].shape)
-    model.loss(
-        model, input_data, outputs["v"], outputs["pi"], pi_mask, multi_counter
-    )  # make sure it computes something
+    pi_mask = torch.ones(outputs["pi_logit"].shape)
+    
+    loss(
+        model, input_data, outputs["v"], outputs["pi_logit"], pi_mask, multi_counter
```

Comment:
you're passing a logit to the loss above but it takes a policy `pi`.

Reply:
Softmax it before passing in or something?

Reply:
I'll fix this in a later PR